addons/stock_calendar/tests/test_stock_calendar.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests import common
+from datetime import datetime, timedelta
+from odoo import fields
+
+
+class TestsStockCalendar(common.TransactionCase):
+
+    def setUp(self):
+        super(TestsStockCalendar, self).setUp()
+
+        # Create a partner
+        self.stock_warehouse0 = self.ref('stock.warehouse0')
+        self.purchase_route_warehouse0_buy = self.ref('purchase.route_warehouse0_buy')
+        self.stock_picking_type_out = self.ref('stock.picking_type_out')
+        self.stock_location = self.ref('stock.stock_location_stock')
+        self.stock_location_customer = self.ref('stock.stock_location_customers')
```

Comment:
yes change variable name with suffix _id.. so we can easily identify and it's readable.